coreneuron/permute/cellorder2.cpp

```diff
@@ -194,61 +159,6 @@ static void move_nodes(size_t start, size_t length, size_t dst, VTN& nodes) {
     }
 }
 
-// least number of nodes to move after nd to eliminate prace
-static size_t need2move(TNode* nd) {
-    size_t d = dist2child(nd);
-    return warpsize - ((nd->nodevec_index % warpsize) + d);
-}
-
-#if DEBUG
```

Comment:
@olupton : can I suggest to put `permute/cellorder*` related code removal in a separate PR or defer it? the reason is that it has some unused code with some macros like `DEBUG` but I believe there is some useful information in the routines and it will help to understand/debug this part of the code. (@kotsaloscv started it but we will ned some more time + efforts)

Reply:
Sure, I changed it so this PR just moves some `#if DEBUG` around to squash the warnings.